01-Subscription-Security/Readme.md

```diff
@@ -913,6 +932,30 @@ AzSK now supports the Privileged Identity Management (PIM) helper cmdlets. This
                               -Force
       ```
 
+
+ 6. <h4> Extend PIM assignments for expiring assignments (-ExtendExpiringAssignments) </h4>
+	 Use this command to extend PIM eligible assignments that are about to expire in n days
+    <b>Example 1: </b>Extend Owner PIM roles that are to be expired in 10 days. This command runs in an interactive fashion so that you get an opportunity to verify the assignments being extended.
```

Comment:
This command runs in an interactive manner in order to verify the assignments being extended.

Reply:
done